### PR TITLE
Minor fixes to docs for Issue 717

### PR DESCRIPTION
**Pull request by ned1313**
Following up on Issue #717. Made minor fixes to the docs. Most of the comments in the issue have already been resolved by other updates.

### CODE REVIEW

File: docs/content/guides/integrations/cert_manager/_index.md [hunk 1]
```diff
@@ -58,17 +58,15 @@ aws route53 change-resource-record-sets --hosted-zone-id $ZONEID --change-batch
 
 ### Add a Service
 
-Add a service that will get exposed via gloo. In this document we will use the petclinic spring application. 
+Add a service that will get exposed via Gloo. In this document we will use the petclinic spring application. 
 
 ```shell
-kubectl apply -f https://raw.githubusercontent.com/solo-io/gloo/v0.8.4/example/petclinic/petclinic.yaml
+kubectl apply -f https://raw.githubusercontent.com/solo-io/gloo/master/example/petclinic/petclinic.yaml
```
Comment (kdorosh) [inline docs/content/guides/integrations/cert_manager/_index.md:64]: these versions aren't necessarily the same, and I much prefer to pin copy/paste commands to versioned yaml in the docs

Comment (ned1313) [reply docs/content/guides/integrations/cert_manager/_index.md:64]: No prob. I'll revert it.

**Pull request merged**